docs/_articles/en/v3.x/faq.md

```diff
@@ -110,18 +110,51 @@ Graph Engine builds up the context of the source code in its entirety before it
 
 To learn how, read about [Engine Directives](./en/v3.x/salesforce-graph-engine/working-with-sfge/#add-engine-directives).
 
-## Questions about Interpreting ApexFlsViolationRule results
+### Questions about Interpreting ApexFlsViolationRule results
 
 #### Q: My data operation is already protected though not through a CRUD/FLS check. I'm confident that a CRUD/FLS check is not needed. How do I make the violation go away?
 
-If you determine that the CRUD operation in question is protected by a sanitizer that SFGE doesn’t recognize, you can add an [engine directive](./en/v3.x/salesforce-graph-engine/working-with-sfge/#add-engine-directives) to let SFGE know that the CRUD operation _is_ in fact safe.
+If you determine that the CRUD operation in question is protected by a sanitizer that Graph Engine doesn’t recognize, you can add an [engine directive](./en/v3.x/salesforce-graph-engine/working-with-sfge/#add-engine-directives) to let Graph Engine know that the CRUD operation _is_ in fact safe.
 
 #### Q: I didn’t get any violations. Does this mean my code is secure?
 
 If you didn’t get any violations, one these is a possibility:
 
-1. SFGE did not identify any entry points
-2. SFGE ran into errors for all the entry points identified
+1. Graph Engine did not identify any entry points
+2. Graph Engine ran into errors for all the entry points identified
 3. Your code is actually secure
 
 Since #1 and #2 exist, you may still want to manually make sure your code is secure.
+
+### Questions About `OutOfMemory` Errors
+
+#### What Factors Contribute to an `OutOfMemory` Error?
+
+Several factors can degrade Graph Engine’s efficiency and increase the probability of encountering an `OutOfMemory` error.
+
+* With every conditional or method invocation in your code, the number of paths Graph Engine creates increases exponentially.
+* Your OS type, Java setup, and other processes running on your machine can influence the heap space assigned by Java Virtual Machine (JVM).
+
+If Graph Engine’s execution is interrupted, it returns results from the portion of source code that it analyzed. We recommend that you add the `--outfile` parameter to your results in your own separate file.
```

Comment:
> parameter to your results
Looks like a word is missing: "to ___ your results"

"to _pipe_ your results", "to _collect_ your results", "to _capture_ your results"

Reply:
+1

Reply:
capture - done

Reply:
I see that I added "capture" to the file and saved it, but it's not showing up here.